testing/waybar/APKBUILD

```diff
@@ -0,0 +1,36 @@
+# Contributor:
+# Maintainer:
+pkgname="waybar"
+pkgver="0.7.1"
+pkgrel=0
+pkgdesc="Highly customizable Wayland bar for Sway and Wlroots based compositors"
+url="https://github.com/Alexays/Waybar/"
+arch="all"
+license="MIT"
+depends=""
+makedepends="gtkmm3-dev jsoncpp-dev spdlog-dev"
+install=""
+source="Waybar-$pkgver.tar.gz::https://github.com/Alexays/Waybar/archive/$pkgver.tar.gz"
+builddir="$srcdir/Waybar-$pkgver"
+
+build() {
+	meson \
+		--prefix=/usr \
+		--sysconfdir=/etc \
+		--mandir=/usr/share/man \
+		--localstatedir=/var \
+		--buildtype=release \
+		. output
+	ninja -C output
+}
+
+check() {
+	# Replace with proper check command(s)
+	:
```

Comment:
This needs to be fixed or a !check with a small comment telling why there is no check enabled made.